Source/CarthageKit/BuildSettings.swift

```diff
@@ -216,7 +216,9 @@ public struct BuildSettings {
 			pathComponent = "\(configuration)\(effectivePlatformName)"
 		}
 
-		let path = (basePath as NSString).appendingPathComponent(pathComponent)
+		// TODO: could use self["variant"] maybe, is SDK is passed with variant
+		let isUIKitForMac = (self.supportsUIKitForMac.value ?? false) && (self["SDK_NAME"].map { $0.contains("macos") }.value ?? false)
+		let path = (basePath as NSString).appendingPathComponent(isUIKitForMac ? "\(pathComponent)-uikitformac" : pathComponent)
```

Comment:
It might be better to split Catalyst support into a separate PR?

Reply:
I'll check and see how much I have tied the two together